docs/DeveloperGuide.md

```diff
@@ -190,6 +206,7 @@ Diet Manager aims to achieve the following:
 |v1.1| student|add food items to the database|do not have to constantly check food nutritional value for common foods|
 |v1.2| student|save my diet history|have a record of my daily food intake|
 |v1.2| student|import my diet history|have access to previous records and be able to progress from there|
+|v1.2| student|export my diet history|can view my previous records on other devices and never ever lose my progress|
 |v1.2| student|receive workout advice based on my excess calorie intake for the day|can maintain my calories for the day|
 |v2.0| student|generate a recommended food plan|know what to eat to meet recommended caloric intake|
 
```

Comment:
Good to go. thks for proofreading the DG

Reply:
ahahah, no problem! thanks for approval!